samples/sampler/stories/qa/DayPicker.js

```diff
@@ -3,6 +3,7 @@ import Heading from '@enact/sandstone/Heading';
 import Item from '@enact/sandstone/Item';
 import Scroller from '@enact/sandstone/Scroller';
 import {Component} from 'react';
+import {number} from '@enact/storybook-utils/addons/controls';
```

Comment:
As I was saying, would you please consider other types of controls for `selected`?
Maybe.. select?

Reply:
If you want to test "number" type only, it seems okay.

Reply:
🤔 I'm not sure. It seems okay to test only for the number type if we titled it well.
But if the case can be tested via Screenshot, do we need this sampler for sure?

Reply:
If you are unsure, I will remove the entire code change of qa-sampler.

Reply:
🆗